cupsfilters/ghostscript.c

```diff
@@ -124,22 +124,22 @@ header_to_gs_args(gs_page_header *h,
       outformat == CF_FILTER_OUT_FORMAT_APPLE_RASTER)
   {
     if (outformat != CF_FILTER_OUT_FORMAT_APPLE_RASTER &&
-	(h->MediaClass[0] |= '\0'))
+	(h->MediaClass[0] != '\0'))
```

Comment:
The code is equivalent but the original version is highly non-idiomatic. ORing something with 0 would leave the value unchanged and return true if and only if the value is not zero. I'm sure the comparison what meant here.